hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OmSnapshotManager.java

```diff
@@ -397,11 +397,32 @@ private static CodecRegistry createCodecRegistryForSnapDiff() {
   }
 
   /**
-   * Get snapshot instance LRU cache.
-   * @return LoadingCache
+   * Get snapshot instance LRU cache size. Used in tests.
```

Comment:
nit: no need to add "Used in tests." in comment. `@VisibleForTesting` is already there and should be sued rather than a comment.